resolver/rewriter_resolver_test.go

```diff
@@ -53,6 +58,7 @@ var _ = Describe("RewriterResolver", func() {
 
 	When("has rewrite", func() {
 		var request *model.Request
+		var fqdnEmptyResponse bool
```

Comment:
I think `expectEmptyResponse` would be clearer since this changes the behavior for the expect and isn't related to the test's FQDNs.

Reply:
Changed it to `expectNilAnswer` since that is what we are checking for in `AfterEach`.